src/Time/_TimeItem.js

```diff
@@ -250,6 +251,10 @@ TimeItem.propTypes = {
     downButtonProps: PropTypes.object,
     format12Hours: PropTypes.bool,
     inputProps: PropTypes.object,
+    localizedText: CustomPropTypes.i18n({
```

Comment:
Could the `Time` component pass these props to `TimeItem` so that consumers only have to pass the localized text once?

Reply:
@greg-a-smith for suggested comment associated with similar suggestion in timepicker

Reply:
To @jbadan's point, the `TimeItem` component is not exported for consumers to use.  It only exists as an internal component to better architect the code.  To that end, the `TimeItem` component does not need the same level of prop type validation that `Time` does.  You could simply make this:
```
localizedText: PropTypes.object.isRequired
```
And then remove it from `defaultProps` below.